spec/lib/open_food_network/feature_toggle_spec.rb

```diff
@@ -46,11 +57,13 @@ def stub_foo(value)
         let(:users) { [user.email] }
 
         before do
-          FeatureToggle.enable(:foo) { |user| users.include?(user.email) }
+          FeatureToggle.enable(:foo) { |user| users.include?(user&.email) }
```

Comment:
I don't get why we need the safe navigation operator here.

Reply:
As explained above, `user` can now be `nil` if `current_user` is `nil`.